code/game/machinery/hologram.dm

```diff
@@ -113,6 +113,10 @@ Possible to do for anyone motivated enough:
 		return ..()
 
 /obj/machinery/holopad/AltClick(mob/living/carbon/human/user)

```

Comment:
oh it is, but it doesn't have any affect since nothing in the proc assumes a specific type